### DISCUSSION TITLE
Additional distributions

### DISCUSSION THREAD

**Discussion by nsteins** [StatMixedML/LightGBMLSS#30] [Q&A]
Thank you for your work on this package, I am finding it very useful. I have some questions on how I might go about adding additional distributions to the modeling process:

- What would be necessary to add a Bernoulli distribution to the package? I've made some initial attempts using pytorch's Bernoulli and replicating other lgbmlss distribution objects, but have not had success. I suspect this may be related to the response function? (since p is bounded to [0,1]) But I don't really follow how those functions are applied
- What are the constraints necessary to use linear combinations of distributions. Say I model a function that I suspect has the form of y = n(x) * b(x) where n has a gaussian distribution and b has a beta distribution. Is it possible to use a combined distribution like this in lgbmlss? What constraints would need to be true for this to be effective?

**Comment by StatMixedML** [comment]
Thanks for your interest in the project. I am on vacation until end of October, so please expect some delay in my reply.
